packages/aragon-cli/package.json

```diff
@@ -1,188 +1,189 @@
-    "ignore": "^4.0.6",
-    "inquirer": "^7.0.0",
-    "internal-ip": "^5.0.0",
-    "listr": "^0.14.3",
-    "listr-input": "0.2.0",
-    "listr-silent-renderer": "^1.1.1",
-    "listr-update-renderer": "^0.5.0",
-    "listr-verbose-renderer": "^0.6.0",
-    "mkdirp": "^0.5.1",
-    "ncp": "^2.0.0",
-    "open": "^7.0.0",
-    "public-ip": "^3.1.0",
-    "rimraf": "^2.6.2",
-    "semver": "^6.1.1",
-    "source-map-support": "^0.5.11",
-    "tmp-promise": "^2.0.1",
-    "truffle": "5.1.4",
-    "truffle-flattener": "^1.2.9",
-    "web3": "^1.2.4",
-    "web3-utils": "^1.2.4",
-    "which": "^2.0.1",
-    "yargs": "git://github.com/yargs/yargs.git#fe380cd356aa33aef0449facd59c22cab8930ac9"
-  },
-  "devDependencies": {
-    "@aragon/apps-agent": "^2.0.0",
-    "@aragon/apps-finance": "^3.0.0",
-    "@aragon/apps-payroll": "^1.0.0",
-    "@aragon/apps-survey": "^1.0.0",
-    "@aragon/apps-token-manager": "^2.0.0",
-    "@aragon/apps-vault": "^4.0.0",
-    "@aragon/apps-voting": "^2.0.0",
-    "@babel/cli": "^7.1.2",
-    "@babel/core": "^7.1.2",
-    "@babel/preset-env": "^7.1.0",
-    "@babel/register": "^7.0.0",
-    "ava": "^2.1.0",
-    "babel-eslint": "^10.0.1",
-    "babel-plugin-istanbul": "^5.2.0",
-    "core-js": "^3.1.4",
-    "eslint": "^6.0.1",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-config-standard": "^14.1.0",
-    "eslint-plugin-ava": "^9.0.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-node": "^10.0.0",
-    "eslint-plugin-prettier": "^3.0.0",
-    "eslint-plugin-promise": "^4.0.1",
-    "eslint-plugin-standard": "^4.0.0",
-    "ganache-cli": "^6.7.0",
-    "nyc": "^14.1.1",
-    "prettier": "^1.15.3",
-    "proxyquire": "^2.1.0",
-    "sinon": "^7.3.1"
-  },
-  "engines": {
-    "node": ">=10.0.0"
-  },
-  "aragon": {
-    "clientRepo": "https://github.com/aragon/aragon",
-    "clientVersion": "775edd606333a111eb2693df53900039722a95dc",
-    "clientPort": "3000",
-    "defaultGasPrice": "2",
-    "defaultIpfsTimeout": "300000"
-  },
-  "ava": {
-    "require": [
-      "@babel/register"
-    ],
-    "files": [
-      "test/**/*.test.js"
-    ]
-  },
-  "nyc": {
-    "include": [
-      "src/**/*.js",
-      "scripts/**/*.js"
-    ],
-    "sourceMap": false,
-    "instrument": true
-  },
-  "babel": {
-    "presets": [
-      [
-        "@babel/preset-env",
-        {
-          "useBuiltIns": "usage",
-          "corejs": "3",
-          "targets": {
-            "node": true
-          }
-        }
-      ]
-    ],
-    "env": {
-      "test": {
-        "plugins": [
-          "istanbul"
-        ]
-      }
-    }
-  },
-  "eslintConfig": {
-    "env": {
-      "es6": true,
-      "node": true
-    },
-    "extends": [
-      "standard",
-      "plugin:ava/recommended",
-      "plugin:prettier/recommended"
-    ],
-    "parser": "babel-eslint"
-  },
-  "prettier": {
-    "singleQuote": true,
-    "semi": false,
-    "trailingComma": "es5",
-    "bracketSpacing": true
-  }
+	"name": "@aragon/cli",
```

Comment:
Any reason for converting this spaces into tabs?

Reply:
Mmm no,we should update that 😅